integration/config/core.yaml

```diff
@@ -390,8 +390,8 @@ peer:
             name: DefaultDecorator
           -
           # FPC Addition:1
-            name: ERCCDecorator
-            library: ../../ercc/ercc-decorator.so
+          #  name: ERCCDecorator
```

Comment:
We should mark the stuff we currently (and temporarily) commented out via a corresponding TODO to mark what we later will have to fix once the plugins and tlcc are integrated ...

Reply:
This still applies to the updated version!